src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,133 @@
+  "name": "70969f8f-eab5-4db7-87bf-5c8053431a85",
+  "permissions": [
+    {
+      "actions": [
+        "*"
+      ],
+      "dataActions": [],
+      "notActions": [
+        "Microsoft.Authorization/elevateAccess/Action"
+      ],
+      "notDataActions": []
+    }
+  ],
+  "roleName": "azure-operator",
+  "roleType": "CustomRole",
+  "type": "Microsoft.Authorization/roleDefinitions"
+}
+```
+
+#### 3. Service Principal
+
+Create the service principal
+
+```
+$ az ad sp create-for-rbac --name "azure-operator-sp" --role "azure-operator" --query '{client_id:appId, secret_key:password, tenant_id:tenant, subscription_id:""}'
+{
+    "client_id": "72bc3de4-3cf8-46c5-bd2b-243368ed0622",
+    "secret_key": "d6b2cb93-cae9-44b3-8ec5-dc5feb8c28ba",
+    "subscription_id": null,
+    "tenant_id": "31f75bf9-3d8c-4691-95c0-83dd71613db8"
+}
+```
+
+## Configure the Giant Swarm organization
+
+In the previous section, we explained how to create the service principal in the Azure subscription in order to run Giant Swarm guest clusters.
+
+Giant Swarm guest clusters are owned by organizations, which allows to control access to clusters, since only members of the owner organization have access to the management functions of a cluster.
+
+In order to run a guest cluster in your Azure subscription, the organization owning your cluster has to know about the service principal you just created.
+
+In the output from step 3 of the previous section you need to replace the `null` value in front of `subscription_id:` with your actual subscription ID. This document is the credential which needs to be registered within Giant Swarm API.
+
+```
+{
+    "client_id": "72bc3de4-3cf8-46c5-bd2b-243368ed0622",
+    "secret_key": "d6b2cb93-cae9-44b3-8ec5-dc5feb8c28ba",
+    "subscription_id": "6ec148b8-8bea-4dd3-82bc-1787c8260e4a",
+    "tenant_id": "31f75bf9-3d8c-4691-95c0-83dd71613db8"
+}
+```
+
+- If you have direct access to the Giant Swarm API, please follow the [documentation](https://docs.giantswarm.io/api/#operation/addCredentials) to set the credentials of your organization via the API.
+
+```
+$ curl -X POST -H "Authorization: giantswarm exampleToken" https://api.g8s.example.westeurope.azure.gigantic.io/v4/organizations/giantswarm/credentials -d \
+'{
+  "provider": "azure",
+  "azure": {
+    "credential": {
+      "client_id": "72bc3de4-3cf8-46c5-bd2b-243368ed0622",
+      "secret_key": "d6b2cb93-cae9-44b3-8ec5-dc5feb8c28ba",
+      "subscription_id": "6ec148b8-8bea-4dd3-82bc-1787c8260e4a",
+      "tenant_id": "31f75bf9-3d8c-4691-95c0-83dd71613db8"
+    }
+  }
+}'
+{"code":"RESOURCE_CREATED","message":"A new set of credentials has been created with ID 'hdc2m1'"}
+```
+- In case you work with a Giant Swarm partner, it might be that you don’t have access to the Giant Swarm API. In that case, please hand over the credential to your partner contact.
```

Comment:
`Giant Swarm partner` - do we have a consensus on this term? I've not heard it before

Reply:
No, it's not an official term. I used it in the AWS BYOC article to talk about a business partner. Can it be understood though?

Reply:
I understood it. Was just curious if this was an *official* thing